### ISSUE TITLE
LOSS Functions

### ISSUE THREAD

**Issue by JYT-byte** [XXYZB/VSNet#5]
Hello, can you disclose the code that the loss function during training mentioned in the article? thank you very much!
